tools/editor/scene_tree_dock.cpp

```diff
@@ -1564,30 +1564,28 @@ static bool _is_node_visible(Node* p_node) {
 
 static bool _has_visible_children(Node* p_node) {
 
-	bool collapsed = p_node->has_meta("_editor_collapsed") ? (bool)p_node->get_meta("_editor_collapsed") : false;
+    bool collapsed = p_node->is_displayed_folded();
+
 	if (collapsed)
 		return false;
 
 	for(int i=0;i<p_node->get_child_count();i++) {
-
 		Node* child = p_node->get_child(i);
+
```

Comment:
Please avoid unnecessary line changes, they make the diff harder to read while they are not related to the functional change.